firestore.rules

```diff
@@ -1,5 +1,7 @@
 rules_version = '2';
 
+// TODO: Add appropriate data structures/rules based on + how we decide we want to lay things out: https://github.com/firebase/functions-samples/blob/master/instagram-auth/database.rules.json
```

Comment:
See the following thread for seeking ideas on how to lay this out 🔒 https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1623754294143800